source/is-tuple.d.ts

```diff
@@ -0,0 +1,71 @@
+import type {IfAny} from './if-any';
+import type {IfNever} from './if-never';
+import type {UnknownArray} from './unknown-array';
+
+type IsTupleOptions = {
```

Comment:
Should be exported.

Reply:
Addressed in [da2888](https://github.com/sindresorhus/type-fest/pull/1024/commits/da28883e7d74799b7d4caabc37f08c36ffe72c05).